buildpipeline/DotNet-CoreFx-Trusted-Linux.json

```diff
@@ -224,7 +225,7 @@
       "continueOnError": false,
       "alwaysRun": false,
       "displayName": "Create Helix Test Jobs",
-      "condition": "ne(variables['PB_EnableCloudTest'], 'false')",
+      "condition": "and(ne(variables['PB_EnableTestBuild'], 'false')), ne(variables['PB_EnableCloudTest'], 'false'))",
```

Comment:
Incorrect # of parenthesis (one extra after first ne clause)

```
and(ne(variables['PB_EnableTestBuild'], 'false'), ne(variables['PB_EnableCloudTest'], 'false'))
```

Reply:
Thanks for spotting that